dlp/src/test/java/dlp/snippets/RedactTests.java

```diff
@@ -16,6 +16,7 @@
 
 package dlp.snippets;
 
+// [START dlp_redact_image]
```

Comment:
Is this intentional? If it is there's no end tag.

Reply:
It was not intentional. Fixed.